src/zcash/Address.cpp

```diff
@@ -179,7 +187,11 @@ std::optional<RecipientAddress> UnifiedAddress::GetPreferredRecipientAddress(
     std::optional<RecipientAddress> result;
     for (const auto& receiver : *this) {
         examine(receiver, match {
-            [&](const OrchardRawAddress& addr) { if (nu5Active) result = addr; },
+            [&](const OrchardRawAddress& addr) {
+              if (nu5Active) {
+                  result = addr;
+              }
```

Comment:
I would prefer
```c++
            [&](const OrchardRawAddress& addr) {
                if (nu5Active) { result = addr; }
            },
```
if possible. Try and see whether `AllowShortIfStatementsOnASingleLine: WithoutElse` does that.

Also why is the indent for a lambda body only 2?